planemo/commands/cmd_workflow_lint.py

```diff
@@ -23,5 +29,6 @@ def cli(ctx, paths, **kwds):
         ctx,
         paths,
         lint_args,
+        iwc_grade=kwds.get("iwc", False)
```

Comment:
Would it be "better" to add `"iwc"` as a new key to `lint_args`?

Reply:
> I tried this but lint_args is shared between tools and workflows.

Yes, you should not modify `build_lint_args`, but you could do this outside of this function. Tool linter also does this, i.e. it adds an `extra_modules` key [here](https://github.com/galaxyproject/planemo/blob/9ab4982d92f910aa76af667cffeb829ff5900424/planemo/tool_lint.py#L31).

Reply:
True, I have not thought about it. Let me change this.